drivers/input/keyboard/cypress-touchkey.c

```diff
@@ -93,8 +93,6 @@ static int touchkey_keycode[5] = {0,KEY_MENU , KEY_HOME, KEY_BACK, KEY_SEARCH};
 	|| defined (CONFIG_USA_MODEL_SGH_T769) || defined(CONFIG_USA_MODEL_SGH_I577) || defined(CONFIG_CAN_MODEL_SGH_I577R)\
 	|| defined(CONFIG_USA_MODEL_SGH_I757) || defined(CONFIG_CAN_MODEL_SGH_I757M)
 static int touchkey_keycode[5] = {0,KEY_MENU , KEY_HOMEPAGE, KEY_BACK, KEY_SEARCH};
-#elif defined (CONFIG_JPN_MODEL_SC_05D)
```

Comment:
It will break the soft keys for the device. I simply copied it from Wild Kernel, however I'll readd the code for that soon  and push that commit as well. Squash them if you want it to look cleaner